### PR TITLE
screencopy: give template and screenshot a width

### PR DESCRIPTION
**Pull request by Saviq**
Otherwise a larger screenshots overflows heavily.

### CODE REVIEW

File: mir-ci/mir_ci/robot_libraries/Screencopy.py [hunk 1]
```diff
@@ -125,9 +125,9 @@ def _log_failed_match(self, screenshot, template):
         """Log a failure with template matching."""
         template_img = Image.open(template)
         template_string = (
-            'Template was:<br /><img src="data:image/png;base64,' f'{self._to_base64(template_img)}" /><br />'
+            'Template was:<br /><img width="100%" src="data:image/png;base64,' f'{self._to_base64(template_img)}" /><br />'
```
Comment (hbatagelo) [inline mir-ci/mir_ci/robot_libraries/Screencopy.py:128]:
```suggestion
            'Template was:<br /><img style="max-width: 100%" src="data:image/png;base64,' f'{self._to_base64(template_img)}" /><br />'
```

How about using `max-width`? It won't resize images that already fit, and it also won't cause overflow.

File: mir-ci/mir_ci/robot_libraries/Screencopy.py [hunk 2]
```diff
@@ -125,9 +125,9 @@ def _log_failed_match(self, screenshot, template):
...
         )
-        image_string = 'Image was:<br /><img src="data:image/png;base64,' f'{self._to_base64(screenshot)}" />'
+        image_string = 'Image was:<br /><img width="100%" src="data:image/png;base64,' f'{self._to_base64(screenshot)}" />'
```
Comment (hbatagelo) [inline mir-ci/mir_ci/robot_libraries/Screencopy.py:130]:
```suggestion
        image_string = 'Image was:<br /><img style="max-width: 100%" src="data:image/png;base64,' f'{self._to_base64(screenshot)}" />'
```

**Review by hbatagelo — CHANGES_REQUESTED** [review]
Looks good, but the template looks huge at width=100%. I'd prefer images that don't overflow not to be scaled up. I've made a suggestion to account for that. WDYT?

**Review by Saviq — COMMENTED** [review]
> Looks good, but the template looks huge at width=100%. I'd prefer images that don't overflow not to be scaled up. I've made a suggestion to account for that. WDYT?

You're right, of course. Thanks!

**Review by hbatagelo — APPROVED** [review]

**Pull request merged**